zod/src/zod.ts

```diff
@@ -63,79 +79,237 @@ function parseErrorSchema(
   return errors;
 }
 
+function parseZod4Issues(
+  zodErrors: z4.$ZodIssue[],
+  validateAllFieldCriteria: boolean,
+) {
+  const errors: Record<string, FieldError> = {};
+  // const _zodErrors = zodErrors as z4.$ZodISsue; //
+  for (; zodErrors.length; ) {
```

Comment:
Interesting syntax, I guess it's the same as `while`, wonder is it just perf benefit?